src/main/kotlin/com/chattriggers/ctjs/triggers/StepTrigger.kt

```diff
@@ -37,15 +37,17 @@ class StepTrigger(method: Any, loader: ILoader) : Trigger(method, TriggerType.St
     }
 
     override fun trigger(args: Array<out Any?>) {
+        val currentTime = args.firstOrNull() as Long? ?: return
```

Comment:
This will always have a long in the first position, so no need to handle anything else:
```suggestion
        val currentTime = args[0] as Long
```